### ISSUE TITLE
External link component missing visually hidden 'new tab' message for screen readers (WCAG Level A)

### ISSUE THREAD

**Issue by jrbarnes9** [ONSdigital/design-system#2145]
External link component missing visually hidden 'new tab' message for screen readers

See `screenreaderMessage` used in [lists](https://ons-design-system.netlify.app/foundations/typography/#lists)

_Originally posted by @jrbarnes9 in https://github.com/ONSdigital/design-system/discussions/233#discussioncomment-2614687_

**Comment by yatesn** [comment]
@jrbarnes9 ok, good to know. So that would push it up our priority list?
